greek/offer_marshmallow_a_cigar/set_it_on_fire/stay_inside/stay_inside.md

```diff
@@ -3,4 +3,10 @@
 κεφάλι σου απο την στενή τρύπα στον τοίχο νιώθεις ένα χέρι να σε
 πιάνει απο το πόδι και αμέσως πετάγεσε στο πίσω μέρος του δωματίου.
 Ελαφρώς ζαλισμένος και με ελαφριά θολή όραση απο το βίαιο ταρακούνημα 
-αντιλαμβάνεσαι οτι στο δωμάτιο βρίσκονται και άλλοι σαν εσένα. 
\ No newline at end of file
+αντιλαμβάνεσαι οτι στο δωμάτιο βρίσκονται και άλλοι σαν εσένα. 
+
+Τι κάνεις;
+
+[Αποφασίζεις να μοιραστείς αυτό που είδες με τους άλλους και μαζί να βρείτε μια λύση.](collaborate_to_take_it_down/collaborate_to_take_it_down.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.